styles/_theme.scss

```diff
@@ -331,7 +331,7 @@ input[type=range][class~="clrS"] {
 }
 
 .chatOpen #appFrame #chatContainer {
-  background-color: $primaryGhost;
+  background-color: $primary;
```

Comment:
If this is removed, you can also remove the $primaryGhost variable from the _variables.scss file, and the .clrPG class from the theme, since the chat container is the only place primaryGhost is currently used.